searchlib/src/vespa/searchlib/features/tensor_attribute_executor.cpp

```diff
@@ -17,6 +18,11 @@ TensorAttributeExecutor(const search::tensor::ITensorAttribute *attribute)
 void
 TensorAttributeExecutor::execute(uint32_t docId)
 {
+    using DTA = search::tensor::DirectTensorAttribute;
```

Comment:
To avoid type checking in the inner-loop, consider creating a separate executor instead and choose that in createTensorAttributeExecutor() in attributefeature.cpp.

Reply:
done.